flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/serializer/jsondebezium/JsonDebeziumChangeContext.java

```diff
@@ -82,8 +84,9 @@ public String getTargetDatabase() {
         return targetDatabase;
     }
 
+    @Deprecated
     public Map<String, String> getTableProperties() {
-        return tableProperties;
+        return Objects.nonNull(dorisTableConfig) ? dorisTableConfig.getTableProperties() : null;
```

Comment:
Will returning null cause problems for downstream users of this method, or can you consider returning an empty map?